ui/admin/app/index.html

```diff
@@ -6,6 +6,7 @@
     <title>Admin</title>
     <meta name="description" content="Simple and secure remote access with HashiCorp Boundary.">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#f24c53">
```

Comment:
I do not have a strong opinion, but thinking loud to get your thoughts. 

Can we use a env var to define this color? I think we are just using it here, so then makes not much sense to create a env var. But if this color is something we will end up defining in other places, would be easier for maintenance to define it in one place.

Reply:
@calcaide long-term, yes we should factor out hardcoded content.  This file is one exception and we can revisit it in the future.